privacyidea/api/lib/postpolicy.py

```diff
@@ -645,3 +645,24 @@ def autoassign(request, response):
                                 break
 
     return response
+
+def construct_radius_response(request, response):
+    """
+    This decorator implements the /validate/radiuscheck endpoint.
+    In case this URL was requested, a successful authentication
+    results in an empty response with a HTTP 204 status code.
+    An unsuccessful authentication results in an empty response
+    with a HTTP 400 status code.
+    :return:
+    """
+    if request.url_rule.rule == '/validate/radiuscheck':
```

Comment:
This is always /validate/check? Even if privacyIDEA runs in a subdirectory like /pi/validate/check?